hasjob/models/jobpost.py

```diff
@@ -181,6 +181,10 @@ class JobPost(BaseMixin, db.Model):
     def get(cls, hashid):
         return cls.query.filter_by(hashid=hashid).one_or_none()
 
+    @classmethod
+    def get_by_ids(cls, ids):
+        return cls.query.filter(cls.id.in_(ids)).all()
```

Comment:
You don't want the entire content of the jobpost here. Use `.options(db.load_only(column_name, ...))` to limit what you load.